image_provisioner/playbooks/roles/docker-config/tasks/main.yaml

```diff
@@ -12,16 +21,61 @@
 - name: install docker
   yum: name=docker state=latest
 
-- name: check for existing PV
-  command: pvdisplay
-  register: pv_result
+- block:
+   # should be used until the mirrors are fixed to install right container-selinux package
+   - name: remove existing container-selinux package
+     yum: name=container-selinux state=absent
+
+   - name: install container-selinux-1.12.5-14
+     yum: name=container-selinux-1.12.5-14.el7 state=latest
```

Comment:
I'd rather not actually commit hard-coding the container-selinux version.   Would prefer people carry it as a local patch for a (hopefully) short period of time